src/App.js

```diff
@@ -84,6 +84,47 @@ export default function App() {
     }
   });
 
+  function success(pos) {
+    var crd = pos.coords;
+    const latitude=crd.latitude;
+    const longitude=crd.longitude;
+    var apikey = '91e34486d51b4b55b4b54c04c205ec80';
```

Comment:
I would 'let' in place of 'var' for the reasons of block scoping and in general as a nice practice